clusterloader2/testing/load/config.yaml

```diff
@@ -317,7 +327,8 @@ steps:
       namespaces: {{$namespaces}}
       tuningSet: RandomizedDeletionTimeLimited
       testMaxReplicaFactor: {{$RANDOM_SCALE_FACTOR}}
-      daemonSetImage: k8s.gcr.io/pause:3.1
+      daemonSetImage: {{$latencyPodImage}}
+      daemonSetEnv: "after update"
```

Comment:
Why do we need to set it here?

Reply:
Personally I'm not fan of not setting it here as this requires custom logic in yaml parsing (https://github.com/kubernetes/perf-tests/blob/a81c576d431247523807fd85caa2ce273db5c7f7/clusterloader2/pkg/config/template_provider.go#L90), but to be consistent with the code near, I removed this.